people/models.py

```diff
@@ -13,32 +16,43 @@
 class ContributorListPage(Page):
     subpage_types = ['ContributorPage']
 
-    people_per_page = models.IntegerField(default=20)
+    def get_rows(self, contributors):
+        rows = []
+        number_of_items = len(contributors)
+        number_of_columns = 3
+        number_of_rows = number_of_items / number_of_columns
```

Comment:
I think here you want 
number_of_rows = number_of_items // number_of_columns
To get the floor of the division.